woocommerce/woocommerce
# Add purchase task E2E test

### All Submissions:






### Changes proposed in this Pull Request:

Adds new purchase task E2E test in order for this to work I had to fix the `admin-e2e-tests` dependency and point it to our local workspace now, and I also added the WooCommerce reset plugin.

Closes #32326 .

### How to test the changes in this Pull Request:

1. Make sure the E2E tests pass in the Github action
2. Pull this branch and run the E2E locally by following these instructions - https://github.com/woocommerce/woocommerce/blob/trunk/plugins/woocommerce/tests/e2e/README.md the whole test suite take a while so you could just run the new tests -> `pnpm exec wc-e2e test:e2e-dev ./tests/e2e/specs/admin-tasks/purchase.test.js` (inside the `plugins/woocommerce` folder)

### Changelog entry

> Add E2E test for the purchase task in the task list.

### FOR PR REVIEWER ONLY:
